galley/pkg/kube/converter/converter.go

```diff
@@ -63,118 +63,144 @@ func Get(name string) Fn {
 	return fn
 }
 
-func identity(_ *Config, destination resource.Info, name resource.FullName, u *unstructured.Unstructured) ([]Entry, error) {
-	p, err := toProto(destination, u.Object["spec"])
-	if err != nil {
-		return nil, err
+func identity(_ *Config, destination resource.Info, name resource.FullName, _ string, u *unstructured.Unstructured) ([]Entry, error) {
+	var p proto.Message
+	var err error
```

Comment:
nit: move `var err error` before line 71. `err` isn't needed outside the `if` conditional scope.

Reply:
... and on line 122.